components/permit-holders/modals/MedicalHistoryModal.tsx

```diff
@@ -57,44 +57,44 @@ export default function MedicalHistoryModal(props: MedicalHistoryModalProps) {
             paddingX="4px"
           >
             <Text as="h2" textStyle="display-medium-bold">
-              {`${medicalInformation.disability} (${medicalInformation.certificationDate})`}
+              {`${application.disability} (${application.createdAt})`}
             </Text>
           </ModalHeader>
           <ModalBody paddingTop="0px" paddingBottom="36px" paddingX="4px">
-            {(medicalInformation.affectsMobility ||
-              medicalInformation.mobilityAidRequired ||
-              medicalInformation.cannotWalk100m) && (
+            {(application.affectsMobility ||
+              application.mobilityAidRequired ||
+              application.cannotWalk100m) && (
               <HStack paddingBottom="22px" spacing="16px">
-                {medicalInformation.affectsMobility && (
+                {application.affectsMobility && (
                   <Badge backgroundColor="background.informative">{'Affects Mobility'}</Badge>
                 )}
-                {medicalInformation.mobilityAidRequired && (
+                {application.mobilityAidRequired && (
                   <Badge backgroundColor="background.informative">{'Aid Required'}</Badge>
                 )}
-                {medicalInformation.cannotWalk100m && (
+                {application.cannotWalk100m && (
                   <Badge backgroundColor="background.informative">{'Cannot walk > 100m'}</Badge>
                 )}
               </HStack>
             )}
 
             <Divider borderColor="border.secondary" />
 
-            {medicalInformation.aid && (
+            {application.aid && (
               <Box paddingTop="24px">
                 <Text as="h3" textStyle="heading" paddingBottom="20px">
                   {'Mobility Aids:'}
                 </Text>
-                {_renderAidsList(medicalInformation.aid)}
+                {_renderAidsList(application.aid)}
               </Box>
             )}
 
-            {medicalInformation.notes && (
+            {application.notes && (
```

Comment:
I think `application.notes` isn't correct here as I believe this refers to notes on the application itself and not notes relating to medical information but I kept it here as a placeholder. There doesn't seem to be a field for that so I'm not sure what should be used here

Reply:
Yes, this is correct. We don't have medical information notes on a per-application basis. It is best to just leave the `application.notes` here, as it is most likely that the notes attached to an application are related to the medical information